agents/__init__.py

```
@@ -4,4 +4,4 @@
 """
 
 __version__ = "1.0.0"
-__all__ = ["executor", "mutator", "registry"] 
\ No newline at end of file
+__all__ = ["executor", "mutator", "registry"]
```

Pylint (3.3.7)

[error] Undefined variable name 'registry' in __all__

(E0603)